src/inmemory_cache.rs

```diff
@@ -139,18 +176,14 @@ impl InMemoryCache for FoyerCache {
 pub(crate) fn create_block_cache(
```

Comment:
I don't think we need this anymore since we're getting rid of `block_cache_options`.

Reply:
removed